pkg/cluster/bootstrap.go

```diff
@@ -576,3 +580,14 @@ func (c *Cluster) Snapshot(ctx context.Context, config *config.Control) error {
 	}
 	return c.managedDB.Snapshot(ctx, config)
 }
+
+// compareClusterConfig verifies that the config of the joining control plane node coincides with the cluster's config
+func (c *Cluster) compareClusterConfig() error {
+	serverConfig, _ := c.clientAccessInfo.Get("/v1-" + version.Program + "/configServer")
+	localConfig := config.HashConfig(c.config)
```

Comment:
Can we revmove the "Config" suffix from this method call? It's an unnecessary stutter in naming.